ui/client/components/graph/node-modal/editors/expression/ExpressionSuggest.js

```diff
@@ -152,7 +154,9 @@ class ExpressionSuggest extends React.Component {
                          fontSize: 16,
                          fontFamily: "'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', 'source-code-pro', monospace", //monospace font seems to be mandatory to make ace cursor work well,
                          readOnly: inputProps.readOnly
-                       }}/>
+                       }}
+                       onFocus={(_) => this.setEditorFocus(true)}
```

Comment:
There should be `onFocus={this.setEditorFocus(true)}` and definition should look: `setEditorFocus = (focus) => () => {}`. Please do in analogic way with onBlur.

Reply:
Done